test/suites/standard/networking-smoke.robot

```diff
@@ -41,3 +43,29 @@ Expose Hello MicroShift Service Via Route
 Delete Hello MicroShift Route
     [Documentation]    Delete route for cleanup.
     Oc Delete    route/hello-microshift -n ${NAMESPACE}
+
+Resolve Hello MicroShift MDNS
+    [Documentation]    Resolve hello-microshift route via mDNS from the hypervisor/RF runner.
+
+    # Open port 5353 on the hypervisor to be able to resolve mDNS hostnames
+    ${result}=    Run Process
+    ...    sudo firewall-cmd --zone\=libvirt --add-service\=mdns
```

Comment:
This does not work when running the test against localhost (inside the VM running MicroShift). This is a setup step changing the hypervisor, not a part of the test. We have `manage_hypervisor_config.sh` for requirements like this.

Reply:
Moved there